dist/partials/config.html

```diff
@@ -16,6 +16,20 @@ <h5>Wavefront Settings</h5>
         </div>
     </div>
 
+    <div class="gf-form-inline">
+        <div class="gf-form">
+            <span class="gf-form-label width-10">CSP API Token</span>
+        </div>
+        <div class="gf-form max-width-24">
+            <i class="gf-form-label max-width-22 text-center" ng-show="ctrl.cspApiTokenExists">Configured</i>
+            <a class="btn btn-secondary gf-form-btn" style="flex-grow:1" type="submit" ng-click="ctrl.resetCspApiToken()" ng-show="ctrl.cspApiTokenExists">Reset</a>
+            <input type="text" class="gf-form-input max-width-24" ng-hide="ctrl.cspApiTokenExists" ng-model="ctrl.current.jsonData.cspAPIToken"/>
+            <info-popover mode="right-absolute" ng-hide="ctrl.cspApiTokenExists">
+                Paste your CSP API token here. You can find and manage your tokens on your profile page in the Wavefront app
```

Comment:
Please update this throughout:

Paste your VMware Cloud services API token here. You can find and manage your API tokens on your profile page in the Wavefront app.